docs/rules/arazzo/parameters-unique.md

```diff
@@ -0,0 +1,93 @@
+---
+slug: /docs/cli/rules/arazzo/parameters-unique
```

Comment:
A lot of the pages have the slug defined, and using it this way does make for URLs that case-match their filenames - which is important since many new rules use mixed camel and kebab case. It does make it more confusing to preview the site locally, but this should be fixed as a separate concern.